lib/container.js

```diff
@@ -331,6 +331,20 @@ function loadSupportObject(modulePath, supportObjectName) {
   }
   try {
     const obj = require(modulePath);
+
+    if (typeof obj === 'function' && supportObjectName === 'I') {
+      const fobj = obj();
+
+      if (fobj.constructor.name === 'Actor') {
+        const methods = getObjectMethods(fobj);
```

Comment:
Wondering if is it 100% required that people implementing a steps file must use the `actor` object I seem to think that it's possible to implement a steps file without doing that i.e. just return an object, and if so if we've subtly breaking people with 3.0

Reply:
Checked my code was wrong think maybe consumers using actor object is required like documented

Reply:
Good quetsion - technically yes, you can have anything. But without an actor, it doesn't have much sense